docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Common Label and Annotation Recommendations
+---
+
+{% capture overview %}
+Many tools can be used to visualize and manage Kubernetes objects. This goes beyond
+tools such as dashboard or kubectl and includes many tools in the Kubernetes ecosystem.
+To enable interoperability between tools, a common set of labels and annotation
+will allow objects to be described in a common manner that all tools can understand.
+
+In addition to supporting tooling, the common labels and annotations describe
```

Comment:
Common (also used in the title) would seem to imply they’re currently in common use, when I think what the intent is here is to have them become common. I’d recommend replacing:

“the common labels and ...” to
> “the recommended labels and ...”